dbos-config.schema.json

```diff
@@ -4,6 +4,18 @@
   "type": "object",
   "additionalProperties": false,
   "properties": {
+    "name": {
```

Comment:
Should we also add the `appCommands` here?

Reply:
Not yet, not until that section is stable